.travis.yml

```diff
@@ -11,10 +11,6 @@ matrix:
   allow_failures:
    - env: GHCVER=head CABALVER=head
 
-branches:
-  only:
-    - master
```

Comment:
this was here so we don't build twice, the branch and PR:
- https://travis-ci.org/futurice/fum2github/builds/61325572
- https://travis-ci.org/futurice/fum2github/builds/61326371

IMHO totally unnecessary to build feature branches